test/loader/test_link_neighbor_loader.py

```diff
@@ -206,16 +206,17 @@ def test_temporal_heterogeneous_link_neighbor_loader():
 
     # With edge_time:
     edge_time = torch.arange(data['paper', 'paper'].edge_index.size(1))
-    loader = LinkNeighborLoader(
-        data,
-        num_neighbors=[-1] * 2,
-        edge_label_index=('paper', 'paper'),
-        edge_label_time=edge_time,
-        batch_size=32,
-        time_attr='time',
-        neg_sampling_ratio=0.5,
-    )
+    loader = LinkNeighborLoader(data, num_neighbors=[-1] * 2,
+                                edge_label_index=('paper', 'paper'),
+                                edge_label_time=edge_time, batch_size=32,
+                                time_attr='time', neg_sampling_ratio=0.5,
+                                drop_last=True)
     for batch in loader:
+        # check if each seed edge has a different batch
+        assert batch['paper'].batch.max() == 47
```

Comment:
I guess this happens due to `batch_size==32` and `neg_sampling_ratio == 0.5`, right? Can we add a comment for this and check via `batch.max() <= 32 + 16`?

Reply:
good point, updated the assert stmt.